ArduCopter/AP_Arming.cpp

```diff
@@ -437,7 +437,7 @@ bool AP_Arming_Copter::gps_checks(bool display_failure)
 
     // warn about hdop separately - to prevent user confusion with no gps lock
     if (copter.gps.get_hdop() > copter.g.gps_hdop_good) {
-        check_failed(ARMING_CHECK_GPS, display_failure, "High GPS HDOP");
+        check_failed(ARMING_CHECK_GPS, display_failure, "High GPS HDOP(%.2lf>%.2lf)", static_cast<double>(copter.gps.get_hdop()/100.0f), static_cast<double>(copter.g.gps_hdop_good/100.0f));
```

Comment:
How about adding units in, since we have them?
```suggestion
        check_failed(ARMING_CHECK_GPS, display_failure, "High GPS HDOP(%.2lfm>%.2lfm)", static_cast<double>(copter.gps.get_hdop()/100.0f), static_cast<double>(copter.g.gps_hdop_good/100.0f));
```